crates/bevy_tasks/src/task_pool.rs

```diff
@@ -309,6 +336,42 @@ mod tests {
         assert_eq!(count.load(Ordering::Relaxed), 100);
     }
 
+    #[test]
+    fn test_spawn_pollable() {
+        let transform_fn = |i: usize| i * 3;
```

Comment:
It's unclear why you use `usize` throughout this function - in this case, `u32` or even `u8` might be clearer, since these values are only numbers; they're not used for anything other than simple arithmetic and comparison (i.e. no pointer maths/indexing)